peer/brontide.go

```diff
@@ -1917,6 +1924,25 @@ func (p *Brontide) hasChannel(chanID lnwire.ChannelID) bool {
 // channel with the peer to mitigate a dos vector where a peer costlessly
 // connects to us and spams us with errors.
 func (p *Brontide) storeError(err error) {
+	// If we do not have any active channels with the peer, we do not store
+	// errors as a dos mitigation.
+	if !p.haveActiveChannels() {
+		p.log.Trace("no channels with peer, not storing err")
+		return
+	}
+
+	p.cfg.ErrorBuffer.Add(
+		&TimestampedError{Timestamp: time.Now(), Error: err},
+	)
+}
+
+// haveActiveChannels checks if the Brontide instance has any active Lightning
+// network channels that are currently open and not pending.
+//
+// Returns:
+// - true if there is at least one active channel.
+// - false if there are no active channels or all channels are pending.
+func (p *Brontide) haveActiveChannels() bool {
```

Comment:
rename to `hasActiveChannels`. It is very uncommon to use "have" instead of "has" in these scenarios.